specification/app/resource-manager/Microsoft.App/preview/2023-05-02-preview/Builders.json

```diff
@@ -439,6 +447,24 @@
       },
       "description": "The updatable properties of the BuilderResource."
     },
+    "IdentityMappings": {
+      "type": "object",
+      "properties": {
+        "identityResourceId": {
+          "type": "string",
+          "description": "Resource ID of the managed identity."
+        },
+        "containerRegistryServer": {
+          "type": "string",
+          "description": "Login server of the container registry."
+        }
+      },
+      "description": "Model representing an environment variable.",
```

Comment:
Could you please confirm that this description is up to date? If yes, please consider adding a bit more details (unless this kind of description is normal, in which case feel free to ignore)

Reply:
missed updating this. thanks